packages/driver/src/cy/commands/cookies.ts

```diff
@@ -92,6 +73,27 @@ function validateDomainOption (domain: any, commandName: string, log: Log | unde
 }
 
 export default function (Commands, Cypress, cy, state, config) {
+  const getDefaultDomain = () => {
+    const hostname = state('window').location.hostname
```

Comment:
Does this need to be typesafe if the window is unloaded if someone tries to execute a cookie command on the wrong origin?
```suggestion
    const hostname = state('window')?.location.hostname
```